WebAPI fails to start with enabled webapi-hive profile

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/opt/tomcat/webapps/WebAPI/WEB-INF/lib/slf4j-log4j12-1.7.26.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/opt/tomcat/webapps/WebAPI/WEB-INF/lib/log4j-slf4j-impl-2.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
16-Dec-2020 11:45:56.024 SEVERE [localhost-startStop-1] org.apache.catalina.core.ContainerBase.addChildInternal ContainerBase.addChild: start: 
        org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/WebAPI]]
                at org.apache.catalina.util.LifecycleBase.handleSubClassException(LifecycleBase.java:440)
                at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:198)
                at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743)
                at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
                at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
                at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:970)
                at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1840)
                at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
                at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
                at java.lang.Thread.run(Thread.java:748)
        Caused by: java.lang.NoSuchMethodError: org.apache.jasper.xmlparser.ParserUtils.parseXMLDocument(Ljava/lang/String;Ljava/io/InputStream;Z)Lorg/apache/jasper/xmlparser/TreeNode;
                at org.apache.jasper.runtime.TldScanner.scanTld(TldScanner.java:600)
                at org.apache.jasper.runtime.TldScanner.scanJar(TldScanner.java:457)
                at org.apache.jasper.runtime.TldScanner.scanJars(TldScanner.java:694)
                at org.apache.jasper.runtime.TldScanner.scanTlds(TldScanner.java:350)
                at org.apache.jasper.runtime.TldScanner.onStartup(TldScanner.java:239)
                at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5144)
                at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
                ... 10 more
16-Dec-2020 11:45:56.025 SEVERE [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Error deploying web application archive [/opt/tomcat/webapps/WebAPI.war]
        java.lang.IllegalStateException: ContainerBase.addChild: start: org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/WebAPI]]
                at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:747)
                at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
                at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
                at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:970)
                at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1840)
                at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
                at java.util.concurrent.FutureTask.run(FutureTask.java:266)
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
                at java.lang.Thread.run(Thread.java:748)
16-Dec-2020 11:45:56.025 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/opt/tomcat/webapps/WebAPI.war] has finished in [13,705] ms
16-Dec-2020 11:45:56.025 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployDirectory Deploying web application directory [/opt/tomcat/webapps/ROOT]
16-Dec-2020 11:45:56.040 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployDirectory Deployment of web application directory [/opt/tomcat/webapps/ROOT] has finished in [15] ms
16-Dec-2020 11:45:56.041 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
16-Dec-2020 11:45:56.046 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in 13766 ms
```

--- a/pom.xml
+++ b/pom.xml
@@ -1395,6 +1395,10 @@
           <version>${hive-jdbc.version}</version>
           <exclusions>
             <exclusion>
+              <groupId>tomcat</groupId>
+              <artifactId>*</artifactId>
+            </exclusion>
+            <exclusion>
               <groupId>org.apache.tomcat</groupId>
               <artifactId>*</artifactId>
             </exclusion>

--- a/src/main/java/org/ohdsi/webapi/JerseyConfig.java
+++ b/src/main/java/org/ohdsi/webapi/JerseyConfig.java
@@ -36,6 +36,7 @@ import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 import javax.inject.Singleton;
+import javax.ws.rs.ext.RuntimeDelegate;
 
 /**
  *
@@ -47,6 +48,7 @@ public class JerseyConfig extends ResourceConfig implements InitializingBean {
     private String rootPackage;
     
     public JerseyConfig() {
+       RuntimeDelegate.setInstance(new org.glassfish.jersey.internal.RuntimeDelegateImpl());
        EncodingFilter.enableFor(this, GZipEncoder.class);
     }
     

